packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -109,6 +109,12 @@ export function ListViewItem<T>(props: ListViewItemProps<T>) {
 
   let showCheckbox = state.selectionManager.selectionMode !== 'none' && state.selectionManager.selectionBehavior === 'toggle';
   let {visuallyHiddenProps} = useVisuallyHidden();
+  let descriptionId = useSlotId();
+  rowProps['role'] = 'row';
+  rowProps['aria-label'] = item.textValue,
+  rowProps['aria-selected'] = state.selectionManager.selectionMode !== 'none' ? isSelected : undefined,
+  rowProps['aria-rowindex'] = item.index + 1,
+  rowProps['aria-describedby'] = descriptionId;
```

Comment:
These should go into `useListItem` if possible (most are probably in there). The merge conflict must have been confusing haha sorry about that

Reply:
Forgot about that, will do